packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -1,11 +1,28 @@
 <h1 id="dedicatedCloud_datacenter_backup_header" data-translate="dedicatedCloud_datacenter_backup_title"></h1>
 <div data-ui-view>
     <div class="datacenter-backup">
-        <div data-ovh-alert="{{ ::$ctrl.alertMain }}"></div>
-        <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+        <div data-ovh-alert="{{ $ctrl.alertMain }}" id="backupMainAlert"></div>
+        <oui-message
+            class="mb-3"
+            data-ng-if="!$ctrl.backup.isActive()"
```

Comment:
I did not get what you are saying. The message is only shown if the backup is not active. Above "data-ng-if" takes care of it.

Reply:
I was wondering why there is no `this.backup.isUnactive()`

Reply:
Inactive means it's not active, it can be in upgrading, removing or in any other state but not in active state. So !$ctrl.backup.isActive() means its inactive.

Reply:
API has no state defined for inactive, it is derived state.